drsyscall/table_windows_ntuser.c

```diff
@@ -781,9 +781,9 @@ syscall_info_t syscall_user32_info[] = {
     {{0,0},"NtUserGetCursorFrameInfo", OK, DRSYS_TYPE_HANDLE, 4,
      {
          {0, sizeof(HCURSOR), SYSARG_INLINED, DRSYS_TYPE_HANDLE},
-         {1, sizeof(DWORD), SYSARG_INLINED, DRSYS_TYPE_UNSIGNED_INT},
-         {2, sizeof(DWORD), R|HT, DRSYS_TYPE_UNSIGNED_INT},
-         {3, sizeof(INT), R|HT, DRSYS_TYPE_SIGNED_INT},
+         {1, sizeof(INT), SYSARG_INLINED, DRSYS_TYPE_SIGNED_INT},
```

Comment:
Hmm, so the ReactOS ntuser.h, which we have in wininc/, was incorrect?
Could you update it as part of this CL:

wininc/ntuser.h:NtUserGetCursorFrameInfo(
wininc/ntuser.h-  HCURSOR hCursor,
wininc/ntuser.h-  DWORD istep,
wininc/ntuser.h-  PDWORD rate_jiffies,
wininc/ntuser.h-  INT *num_steps);

Reply:
Done